pkg/controller/datavolume-controller.go

```diff
@@ -790,8 +790,7 @@ func buildHTTPClient() *http.Client {
 // that 'owns' it.
 func newPersistentVolumeClaim(dataVolume *cdiv1.DataVolume) (*corev1.PersistentVolumeClaim, error) {
 	labels := map[string]string{
-		"cdi-controller": dataVolume.Name,
```

Comment:
So I know these don't look like they are being used, but we need them if we want to change the PVC names to not match the DV names. Tomasz's PR is using these.

Reply:
If he is using this from import/upload/clone controller to find dv then annotation would be easier/better (or even the owner reference)

but if he needs it in datavolume controller to find pvc, then it is not so simple, I need to check that PR

Reply:
He needs it in the datavolume controller to find the PVC, and in some other places like tests. So definitely coordinate because that is currently my issue with this PR, other than that looks pretty good.

Reply:
Okay, we discussed, and we are fine with removing these labels.